app/test_main.py

```diff
@@ -1,3 +1,74 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_age",
+        [
+            (
+                14,
+                14,
+                [0, 0]
+            ),
+            (
+                15, 15, [1, 1]
+            ),
+            (
+                23, 23, [1, 1]
+            ),
```

Comment:
There is no need for such broad spacing, it affects the code perception. Reformat your code style as follows:
```suggestion
        [
            (14, 14, [0, 0]),
            (15, 15, [1, 1]),
            (23, 23, [1, 1]),
```
Apply this to `test_get_human_age_invalid_error` as well.